apps/meteor/client/omnichannel/additionalForms/BusinessHoursMultiple.tsx

```diff
@@ -45,29 +45,29 @@ const BusinessHoursMultiple = ({ className }: { className?: ComponentProps<typeo
 						name='name'
 						control={control}
 						rules={{ required: t('Required_field', { field: t('Name') }) }}
-						render={({ field }) => <TextInput id={nameField} {...field} aria-describedby={`${departmentsField}-error`} />}
+						render={({ field }) => <TextInput id={nameField} {...field} aria-describedby={`${nameField}-error`} />}
 					/>
 				</FieldRow>
 				{errors?.name && (
-					<FieldError aria-live='assertive' id={`${departmentsField}-error`}>
+					<FieldError aria-live='assertive' id={`${nameField}-error`}>
 						{errors.name.message}
 					</FieldError>
 				)}
 			</Field>
 			<Field className={className}>
-				<FieldLabel htmlFor={departmentsField}>{t('Departments')}</FieldLabel>
+				<FieldLabel id={departmentsFieldLabelId}>{t('Departments')}</FieldLabel>
```

Comment:
Why `id` the label instead of the field?

Reply:
This is a temporary work around since `AutoCompleteDepartmentMultiple` (and its underlying component) do not pass the provided id to the input, it passes to a div wrapper. This causes problems when trying to reach the component semantically (getByLabel).  The spec says that `<label>` `for` should point to a [labelable form-related element](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/label#attributes), so div would not be compliant. That's why I went with the aria attribute instead.